apps/graphql-playground/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.2.5",
   "private": true,
   "devDependencies": {
-    "@contentful/app-scripts": "^0.15.8",
+    "@contentful/app-scripts": "^1.2.0",
```

Comment:
Should we also pin this one, similar to other apps?

Reply:
Done!